configure.ac

```diff
@@ -1818,7 +1861,11 @@ AM_CONDITIONAL(ENABLE_DISTCHECK_WORKAROUND, test x$enable_distcheck_workaround =
 if test "x$in_git_src" = "xyes"; then
         AC_MSG_NOTICE([Running from git source])
 
+if test "$unamestr" = "AIX"; then
```

Comment:
This looks wrong. You basically tell `configure` to never ever re-generate man pages on AIX. Even if make detects that this is necessary... so you will run into a make error somewhere in future and can only fix it by hand.